docs/src/main/asciidoc/sql.adoc

```diff
@@ -64,6 +64,7 @@ There is also some configuration specific to Google Cloud SQL:
 | `spring.cloud.gcp.sql.database-name` | Name of the database to connect to. |
 | `spring.cloud.gcp.sql.instance-connection-name` | A string containing a Google Cloud SQL instance's project ID, region and name, each separated by a colon.
 For example, `my-project-id:my-region:my-instance-name`. |
+| `spring.cloud.gcp.sql.ip-types` | Allows you to specify a comma delimited list of preferred IP types for connecting to a Cloud SQL instance. See https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory#specifying-ip-types[Cloud SQL Socket Factory - Specifying IP Types] |
```

Comment:
Can you mention the default `PUBLIC,PRIVATE` here?

Reply:
Done

Reply:
If you look at the preview, there is also a "Default value" column you can populate.
![image](https://user-images.githubusercontent.com/17732492/92933724-9e43fb00-f414-11ea-9194-e755d94d53e6.png)

Reply:
Yes, I saw the "Default value" column, but felt that populating that would signify that spring-cloud-gcp was setting thoose default values. I now see that `spring.cloud.gcp.sql.credentials.location` specifies that it's Spring GCP Boot starter that supplying the defaults.

Suggested rewrite:
| `spring.cloud.gcp.sql.ip-types` | Allows you to specify a comma delimited list of preferred IP types for connecting to a Cloud SQL instance. See https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory#specifying-ip-types[Cloud SQL Socket Factory - Specifying IP Types] |  Default values (`PUBLIC,PRIVATE`) provided by Cloud SQL Socket Factory

Reply:
The default doesn't have to be provided by this project. The column is there to make it more clear what would be the default.